devfiles/java-mysql/devfile.yaml

```diff
@@ -8,7 +8,7 @@ projects:
     source:
       type: git
       location: "https://github.com/che-samples/java-spring-petclinic"
-      commitId: 93873665453584deb891569c02e3acc7283cf394
+      commitId: main
```

Comment:
it should be `branch` instead of `commitId` attribute
or 
real value of the commit Id
I'd prefer the second variant

Reply:
Since we don't update the sample frequently, let's use `branch` property.